docs/cmdline-opts/retry-all-errors.d

```diff
@@ -0,0 +1,19 @@
+Long: retry-all-errors
+Help: Retry all errors (use with --retry) (read manpage, don't use by default)
```

Comment:
I think you should remove the second parenthesis. Users need to read the man page to understand all options!